python/paddle/v2/fluid/tests/test_block_expand_op.py

```diff
@@ -0,0 +1,178 @@
+import unittest
+import numpy as np
+from op_test import OpTest
+
+
+def get_output_shape(attrs, x):
+    img_height = x.shape[2]
+    img_width = x.shape[3]
```

Comment:
I think it's better to pass x.shape instead x.

Reply:
Thx. Fixed.